src/clover/fileparser.py

```diff
@@ -2218,37 +2220,8 @@ def _parse_grid_inputs(
             tiers
         ))
         # WHERE ARE THE EMISSIONS IN PARSE GRID
-    #print (grids)
-    return grids # but we are not returning the tiers?
 
-    # exchange_rate=grid_inputs["exchange_rate"] #why if we ADD [0] this doesn't work
-
-    # # Determine the emissions
-    # grid_emissions = grid_inputs["emissions"]
-    #     # entry [GRID_EMISSIONS]
-    #     # for entry in grid_inputs[GRID]
-    # diesel_5A_costs= grid_inputs["grid"][0]["costs"]
-    # diesel_10A_costs= grid_inputs["grid"][1]["costs"]
-    # edl_threshold_1=grid_inputs["grid"][2]["costs"]
-    # edl_threshold_2=grid_inputs["grid"][3]["costs"]
-    # edl_threshold_3=grid_inputs["grid"][4]["costs"]
-    # edl_threshold_4=grid_inputs["grid"][5]["costs"]
-    # edl_threshold_5=grid_inputs["grid"][6]["costs"]
-    # thresholds=[]
-    # for i in range (2,7):
-    #     thresholds.append(grid_inputs["grid"][i]["lower_bound"])    
-    # return (
-    #     exchange_rate,
-    #     grid_emissions,
-    #     diesel_5A_costs,
-    #     diesel_10A_costs,
-    #     edl_threshold_1,
-    #     edl_threshold_2,
-    #     edl_threshold_3,
-    #     edl_threshold_4,
-    #     edl_threshold_5,
-    #     thresholds,
-    # )
+    return grids # but we are not returning the tiers?
```

Comment:
You've put the tiers on each grid on line 2220 :smile:

Reply:
@BenWinchester I still don't get what should I do here...